source/common/stats/thread_local_store.cc

```diff
@@ -274,8 +285,8 @@ void ThreadLocalStoreImpl::clearScopeFromCaches(uint64_t scope_id,
 
 ThreadLocalStoreImpl::ScopeImpl::ScopeImpl(ThreadLocalStoreImpl& parent, const std::string& prefix)
     : scope_id_(parent.next_scope_id_++), parent_(parent),
-      prefix_(Utility::sanitizeStatsName(prefix), parent.symbolTable()),
-      central_cache_(new CentralCacheEntry(parent.symbolTable())) {}
+      prefix_(Utility::sanitizeStatsName(prefix), parent.alloc_.symbolTable()),
```

Comment:
why do we need the explicit `alloc_.` deref here?

Reply:
making clang_tidy happy.